docs/_utils/deploy.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
+# Clone repo
 git clone "https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git" --branch gh-pages --single-branch gh-pages
-cp -r docs/_build/dirhtml/* gh-pages/
+cp -r docs/_build/dirhtml/* gh-pages
+# Redirect index to master
+cp docs/_utils/redirect.html gh-pages/index.html
```

Comment:
It make more sense to redirect to latest tagged version.
We usually do not want people to use master

Reply:
Or we can make this a variable

Reply:
answered in slack